[#4 #9] Meta tags and Social Share

harshitbudhraja opened the pull request.

hrhv committed:
Changed colours for sharing social buttons

hrhv committed:
Improved social share buttons

hrhv committed:
Added meta tags

rhnvrm commented:
1.  Open sharer in new window.
2.  #LevelUp #Breeze16 in tweets, use @snu_breeze
3. Write the meta tag description, as you think might be appropriate to appear on a google search, current one is not appropriate.
4. Use a better sentence for whatsapp share, one that seems like was written by a human, add link to fb page also in whatsapp share. Use fa icon it has whatsapp http://fontawesome.io/icon/whatsapp/

ghost commented:
@rhnvrm , You can see the code. I have used the fa-whatsapp from fontawesome but it seems not working...I can't figure out why, but I'm working on that :)
In the upcoming pull request:-
1. Open in new page resolved...
2. Tweets improved as requested...
3. I'm not able to figure out the text for the meta tag description and the whatsapp share :p Can we take views from the OC regarding that ?
4. Learn more button is pointed to About us and register to the already existing form

hrhv committed:
Changes as follows

1. Optimised the description for the meta tags 2. hrefs for the register and learn more buttons pointed to the desired locations 3. Tweet now carries the hashtags, @snu_breeze handle, custom text and the website link
